frontend/src/pages/DoctorEditProfilePage.jsx

```
@@ -150,102 +425,295 @@ const DoctorEditProfilePage = () => {
         }
     };
 
-    if (loading) return <div className="text-center p-10 text-white">Loading your profile...</div>;
 
+  // 4. Deactivate logic (copied from the end of your original file)
+  const disable2FA = async () => {
+    /* ... existing implementation ... */
+  };
+  const handleDeactivate = async () => {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'handleDeactivate'. Consider to delete it or rename it.

'handleDeactivate' is defined here:


(lint/suspicious/noRedeclare)